pulsar-flink-connector-shade-2.11/pom.xml

```diff
@@ -0,0 +1,154 @@
+<!--
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>pulsar-flink-parent</artifactId>
+        <groupId>io.streamnative.connectors</groupId>
+        <version>2.7.6-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>pulsar-flink-connector-${scala.binary.version}-${flink.binary.version}</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <flink.version>1.12.1</flink.version>
+        <flink.binary.version>1.12</flink.binary.version>
```

Comment:
```xml
<flink.version>1.12.1</flink.version>
<flink.binary.version>1.12</flink.binary.version>
```

Could be moved to root pom since it was shared by three modules.

Reply:
ok